rct229/rulesets/ashrae9012019/section21/section21rule6.py

```diff
@@ -127,6 +127,28 @@ def rule_check(self, context, calc_vals=None, data=None):
             boiler_2_operation_upper_limit = calc_vals["boiler_2_operation_upper_limit"]
             boiler_2_rated_capacity = calc_vals["boiler_2_rated_capacity"]
 
+            return (
+                boiler_1_operation_lower_limit == ZERO.POWER
+                and boiler_1_operation_upper_limit == boiler_1_rated_capacity
+                and boiler_2_operation_lower_limit == boiler_1_rated_capacity
+                and boiler_2_operation_upper_limit
+                == boiler_1_rated_capacity + boiler_2_rated_capacity
+            ) or (
+                boiler_2_operation_lower_limit == ZERO.POWER
+                and boiler_2_operation_upper_limit == boiler_2_rated_capacity
+                and boiler_1_operation_lower_limit == boiler_2_rated_capacity
+                and boiler_1_operation_upper_limit
+                == boiler_2_rated_capacity + boiler_1_rated_capacity,
+            )
+
+        def is_tolerance_fail(self, context, calc_vals=None, data=None):
+            boiler_1_operation_lower_limit = calc_vals["boiler_1_operation_lower_limit"]
+            boiler_1_operation_upper_limit = calc_vals["boiler_1_operation_upper_limit"]
+            boiler_1_rated_capacity = calc_vals["boiler_1_rated_capacity"]
+            boiler_2_operation_lower_limit = calc_vals["boiler_2_operation_lower_limit"]
+            boiler_2_operation_upper_limit = calc_vals["boiler_2_operation_upper_limit"]
+            boiler_2_rated_capacity = calc_vals["boiler_2_rated_capacity"]
+
             return (
                 boiler_1_operation_lower_limit == ZERO.POWER
```

Comment:
do we need std_equal for Ln153? I saw that in Section 19 you modified the previous code to `std_equal(ZERO.FLOW, return_fans_airflow)` @yunjoonjung-PNNL

Reply:
I believe ```ZERO.POWER``` doesn't need to be compared using ```std_equal``` because this is rather checking whether the value is equal to zero or not.